generate_parameter_library_py/generate_parameter_library_py/string_filters_cpp.py

```diff
@@ -10,13 +10,23 @@ def valid_string_cpp(description):
       str: The filtered string that is a valid C++ string.
     """
     if description:
-        filtered_description = (
-            description.replace('\\', '\\\\').replace('"', '\\"').replace('`', '')
-        )
-        # create a quote delimited string for every line
-        filtered_description = '\n'.join(
-            f'"{line}"' for line in filtered_description.splitlines()
-        )
-        return filtered_description
+        filtered_description = description.replace('\n', '\\n    ')
```

Comment:
Or this should be fine too? This would fix your indent and also the markdown syntax I added to [JTC](https://github.com/ros-controls/ros2_controllers/blob/master/joint_trajectory_controller/src/joint_trajectory_controller_parameters.yaml). 
```suggestion
        # remove possible markdown/rst syntax, but add proper indent for cpp-header files. 
        filtered_description = description.replace('\\', '\\\\').replace('"', '\\"').replace('`', '').replace('\\n', '\\n    ')
```

Reply:
I added this change but made on small change. It should replace `\n' with `\\n' for the new liens to render correctly.